test/basics.jl

```diff
@@ -175,3 +175,10 @@ end
     print(io, GAP.AbstractAlgebra.Lowercase(), GapObj([1, 2, 3]))
     @test String(take!(io)) == "GAP: [ 1, 2, 3 ]"
 end
+
+@testset "versioninfo" begin
+    io = IOBuffer()
+    GAP.versioninfo(io, jll = true)
```

Comment:
```suggestion
    GAP.versioninfo(io; full = true)
```
maybe it's easiest to just test this instead as it enables all code paths

Reply:
o.k.